SerialPrograms/Source/PokemonBDSP/Inference/BoxSystem/PokemonBDSP_BoxNatureDetector.cpp

```diff
@@ -0,0 +1,48 @@
+/*  IV Checker Reader
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "CommonFramework/ImageTypes/ImageViewRGB32.h"
+#include "PokemonSwSh/Inference/PokemonSwSh_IVCheckerReader.h"
```

Comment:
Is this include needed?

Reply:
forgot to remove it, removed with latest commit